geometry_msgs/Point pos
int32 u
int32 v
float32 confidence
